BenchmarkDotNet.Diagnostics/project.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "title": "BenchmarkDotNet.Diagnostics",
+  "frameworks": {
+    "net40": {
+    },
+    "dnx451": {
```

Comment:
Is this a library or an app? `dnx451` is an app-TFM and shouldn't be used for libraries.